tests/fast-integration/installer/helm.js

```diff
@@ -45,6 +45,9 @@ async function helmInstallUpgrade(release, chart, namespace, values, profile) {
     release,
     chart,
   ];
+  if (additionalArgs) {
```

Comment:
Please check if `additinalArgs` is the instance of Array. JS offers you [`Array.isArray()` function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray) for this purpose.